### PR TITLE
Tasks containers tests

### PR DESCRIPTION
**Pull request by eugene-chr**

### CODE REVIEW

File: src/components/Tasks/TaskForm/__tests__/container.spec.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    instance.showManageButtons();
+    expect(instance.state.isShowManageButtons).toBeTruthy();
+  });
+
+  it('hideManageButtons()', () => {
+    instance.hideManageButtons();
+    expect(instance.state.isShowManageButtons).toBeFalsy();
+  });
+
+  it('handleReset()', () => {
```
Comment (plzen) [inline src/components/Tasks/TaskForm/__tests__/container.spec.js]: Add resetForm

File: src/components/Tasks/TaskItem/TaskItemForm/__tests__/container.spec.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    .dive();
+  const instance = container.instance();
+  const spyResetForm = jest.fn();
+
+  it('should match snapshot', () => {
+    expect(container).toMatchSnapshot();
+  });
+
+  it('handleReset()', () => {
+    container.setProps('resetForm', spyResetForm);
```
Comment (plzen) [inline src/components/Tasks/TaskItem/TaskItemForm/__tests__/container.spec.js]: Use spyOn

File: src/components/Tasks/TaskItem/TaskModalComments/TaskModalCommentsList/__tests__/container.spec.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  const props = {
+    comments: [{}],
+    noItems: [{}],
+    loading: false,
+  };
+  const wrapper = shallow(<TaskModalCommentsListComponent {...props} />);
+  const container = wrapper
+    .dive()
+    .dive()
+    .debug();
```
Comment (plzen) [inline src/components/Tasks/TaskItem/TaskModalComments/TaskModalCommentsList/__tests__/container.spec.js]: Remove debug

File: src/components/Tasks/TaskItem/__tests__/__snapshots__/container.spec.js.snap [hunk 4]
```diff
@@ -0,0 +1,91 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`TaskItem container should match snapshot 1`] = `
+<TaskItemComponent
+  commentsCount={0}
+  date="NaN / NaN / NaN"
```
Comment (plzen) [inline src/components/Tasks/TaskItem/__tests__/__snapshots__/container.spec.js.snap]: Use formatted date

File: src/components/Tasks/TaskItem/__tests__/container.spec.js [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+  const wrapper = shallow(<TaskItem {...props} store={store} />);
+  const container = wrapper.dive().dive();
+  const instance = container.instance();
+
+  it('should match snapshot', () => {
+    expect(container).toMatchSnapshot();
+  });
+
+  describe('handleCheckTask()', () => {
+    instance.handleCheckTask();
```
Comment (plzen) [inline src/components/Tasks/TaskItem/__tests__/container.spec.js]: Check

File: src/components/Tasks/TaskList/__tests__/container.spec.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+  const props = {
+    tasks: [{}],
+    noItems: [{}],
+    loading: false,
+  };
+  const wrapper = shallow(<TaskList {...props} />);
+  const container = wrapper
+    .dive()
+    .dive()
+    .debug();
```
Comment (plzen) [inline src/components/Tasks/TaskList/__tests__/container.spec.js]: Remove debug

File: src/components/Tasks/TaskItem/TaskModalComments/TaskModalCommentsList/__tests__/container.spec.js (hunk 3 above)
Comment (eugene-chr) [reply src/components/Tasks/TaskItem/TaskModalComments/TaskModalCommentsList/__tests__/container.spec.js]: done

File: src/components/Tasks/TaskItem/__tests__/__snapshots__/container.spec.js.snap (hunk 4 above)
Comment (eugene-chr) [reply src/components/Tasks/TaskItem/__tests__/__snapshots__/container.spec.js.snap]: done

File: src/components/Tasks/TaskItem/__tests__/container.spec.js (hunk 5 above)
Comment (eugene-chr) [reply src/components/Tasks/TaskItem/__tests__/container.spec.js]: done

File: src/components/Tasks/TaskList/__tests__/container.spec.js (hunk 6 above)
Comment (eugene-chr) [reply src/components/Tasks/TaskList/__tests__/container.spec.js]: done

File: src/components/Tasks/TaskForm/__tests__/container.spec.js (hunk 1 above)
Comment (eugene-chr) [reply src/components/Tasks/TaskForm/__tests__/container.spec.js]: done

**Pull request merged**